grammars/jsdoc.cson

```diff
@@ -411,6 +435,120 @@
         ]
       }
     ]
+  # Default @param value, highlighted as JavaScript
+  'default-value':
+    'patterns': [
+      # Double-quoted string
+      {
+        'begin': '(")'
```

Comment:
@50Wliu If it's really this much of an issue, I'll drop it for you. I'm not going to let this be delayed on something this trivial.

Hold on.